deployment/keystone/changeset/view_contracts.go

```diff
@@ -135,6 +149,49 @@ func GenerateOCR3ConfigView(ocr3Cap ocr3_capability.OCR3Capability) (OCR3ConfigV
 	}, nil
 }
 
+func GenerateForwarderView(ctx context.Context, f *forwarder.KeystoneForwarder) (ForwarderView, error) {
+	// This could be effectively done with 2 other approaches:
+	// 1. Fetching the transaction receipt of the contract deployment, getting the deployment block number,
+	//    and extracting the config from the logs, but we don't have access to the transaction hash needed for this.
+	// 2. Using `CodeAt()` to find the block number in which the contract was created, and use that.
+	//    We would have to go from block number 0 to find it, which in the end is similar what's done here.
+	configIterator, err := f.FilterConfigSet(&bind.FilterOpts{
```

Comment:
Have we validated that this will work with the RPCs we're using? IIUC this would require the use of an archive RPC no?

Reply:
I have already generated the state here https://github.com/smartcontractkit/chainlink-deployments/pull/803/ using these changes @cedric-cordenier